fe/fe-core/src/main/java/com/starrocks/server/GlobalStateMgr.java

```diff
@@ -3195,4 +3076,24 @@ public void doTaskBackgroundJob() {
             LOG.warn("task manager clean expire task runs history failed", t);
         }
     }
+
+    public StateChangeExecution getStateChangeExecution() {
+        GlobalStateMgr gsm = this;
+        StateChangeExecution execution = new StateChangeExecution() {
+            @Override
+            public void transferToLeader(FrontendNodeType newType) {
+                gsm.transferToLeader(newType);
+            }
+
+            @Override
+            public void transferToNonLeader(FrontendNodeType newType) {
+                gsm.transferToNonLeader(newType);
+            }
+        };
```

Comment:
Better to create new execution when global state mgr init.

Reply:
done